views/layout.erb

```diff
@@ -6,7 +6,7 @@
   <link href="<%= stylesheet_path 'main' %>" rel="stylesheet">
 
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
-
+  <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
```

Comment:
The jQuery syntax in `main.js` was a mixup from other JS related experiments. Instead of loading jQuery, you might as well change `main.js`:

```patch
- $(function() {
+ document.addEventListener('DOMContentLoaded', function(){
```